src/elements/otp-generator/otp-generator.html

```diff
@@ -45,8 +49,8 @@
       var that = this;
       new Polymer.IronMeta({type: 'otp', key: 'otp', value: this});
       this.timer = setInterval(function() {
-        var rest = 60 - (new Date()).getSeconds();
-        that.fire('timer', {remaining: rest});
+        var remaining = (60 - (new Date()).getSeconds()) % that.refreshInterval;
```

Comment:
Thanks for pointing them out. That is indeed. I have fixed those 2 problems. Please review.